src/connection/database/login_sign_up_tabs.jsx

```diff
@@ -38,9 +42,16 @@ export default class LoginSignUpTabs extends React.Component {
   }
 
   handleSignUpClick() {
+    if (this.props.signUpLink) {
```

Comment:
Should we consider the `closable` property value with this or should we just ignore it?